docs/configure-and-manage/manage/qlight-node.md

```diff
@@ -73,10 +74,9 @@ The valid values are:
 You can add an encryption layer on the qlight client-server communication.
 Configure the encryption layer using the following options:
```

Comment:
Since the options descriptions are simply repeated here, I recommend changing to:

```suggestion
Configure the encryption layer using the [`qlight.tls.*`](../../reference/cli-syntax.md#qlighttls) command line options.
```

And deleting the bulleted list.